flexget/plugins/urlrewrite_nnmclub.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import requests
```

Comment:
We actually have a requests util, so it should be `from flexget.utils import requests` instead. Sorry :D